tools/cldr-apps/README.md

```diff
@@ -173,6 +176,7 @@ docker compose up cldr-apps
 This will compile and launch the [webdriver](../cldr-apps-webdriver/README.md).
 
 ```shell
+cd tools/cldr-apps
```

Comment:
Could advise running this if running on non-Intel processor such as mac M1/M2/...:

`export DOCKER_DEFAULT_PLATFORM=linux/amd64`

That did fix the problem for me, to prevent an error like `no matching manifest for linux/arm64/v8 in the manifest list entries`

Reply:
If we can just add this to the configuration, that's probably worthwhile, then it ought to work everywhere

Reply:
i just added it as a note for now.